virtual_rainforest/core/base_model.py

```diff
@@ -160,7 +280,57 @@ def __repr__(self) -> str:
 
     def __str__(self) -> str:
         """Inform user what the model type is."""
-        if hasattr(self, "model_name"):
-            return f"A {self.model_name} model instance"
-        else:
-            return "A base model instance"
+        return f"A {self.model_name} model instance"
+
+    def check_required_init_vars(self) -> None:
```

Comment:
Just to be sure I get this: the checks in `init_subclass` are done when setting up the class - i.e. when importing `vr` and these ones are checked when running the simulation, at the config stage, right? 

I would rename this method because it seems a bit confusing checking the `required_init_vars` in two places considering that in one case you are checking the class and in another the contents of data.

Reply:
That's correct on the functionality - and I agree that the naming is ambiguous.